app/views/utils/alert.scala.html

```diff
@@ -3,7 +3,9 @@
 @bootstrapNames = @{
     Map(
         "error" -> "alert-danger",
-        "success" -> "alert-success"
+        "success" -> "alert-success",
```

Comment:
If we now support more alert features, can you provide shorthand methods for them similar to `withError` and the others?